app.js

```diff
@@ -23,6 +23,8 @@ app.use('/chosen', express.static('node_modules/chosen-js'))
 app.use('/dropzone', express.static('node_modules/dropzone/dist'))
 app.use('/jquery-ui', express.static('node_modules/jquery-ui-dist'))
 app.use('/hamburgers', express.static('node_modules/hamburgers/dist'))
+app.use('/bootstrap/css', express.static('node_modules/bootstrap/dist/css'))
+app.use('/tokenize2', express.static('node_modules/tokenize2/dist'))
```

Comment:
Copying and modifying a css dependency works, but it make it hard to tell what was modified, updating the dependency becomes more difficult and it causes copyright/licensing problems.
It is best to use the dependency unmodified (I already implemented that in eb9dba7) and then have a separate css file (styles_general.css ?) with your modifications.
https://developer.mozilla.org/de/docs/Web/CSS/Spezifit%C3%A4t

Reply:
I shortened the stylesheet (removed duplicate) and inserted it into the styles_general.css.